backend-api/src/functions/asyncBiometricToken/validateRequestBody/validateRequestBody.test.ts

```diff
@@ -0,0 +1,79 @@
+import { validateRequestBody } from "./validateRequestBody";
+
+describe("Validate request body", () => {
+  describe.each([
+    [
+      "Given body is null or undefined",
+      null,
+      "Request body is either null or undefined. Request body: null",
+    ],
+    [
+      "Given body cannot be parsed as JSON",
+      "foo",
```

Comment:
Very pedantic nitpick/bugbear: can we use something more meaningful than 'foo' for examples? e.g. 'invalid', 'not JSON', 'arbitrary string', etc.

Reply:
Done